components/camel-slack/src/main/java/org/apache/camel/component/slack/SlackConstants.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.camel.component.slack;
 
+@Deprecated
```

Comment:
These constants before used to build the Json request to Slack APIs, isn't? Is there any reason to deprecate them instead of removing this class entirely?

Reply:
Yes but the class is public instead of package only so I was not sure about removing it. Maybe some used them in their code to do something.

Reply:
okay we can keep it for one version and then remove it